src/MergeContext.js

```diff
@@ -667,23 +658,21 @@ class PullRequest {
         return false;
     }
 
-    // loads info about the "staging tag" (if any); M-staged-PRnnn tag is a
-    // PR-specific git tag pointing to the previously created staged commit
-    async _loadTag() {
-       if (this._tagSha)
-           return;
-
-       try {
-           this._tagSha = await GH.getReference(this._stagingTag());
-       } catch (e) {
-           if (e.name === 'ErrorContext' && e.notFound()) {
-               this._logEx(e, this._stagingTag() + " not found");
-               return;
-           }
-           throw e;
-       }
+    // loads info about the PR staged commit, if any
+    async _loadStaged() {
+        assert(!this._stagedSha());
+        const stagedSha = await GH.getReference(Config.stagingBranchPath());
+        const stagedCommit = await GH.getCommit(stagedSha);
+        const prNum = Util.ParsePrNumber(stagedCommit.message);
+        if (prNum !== null && this._prNumber().toString() === prNum) {
+            this._log("found staged commit " + stagedSha);
+            this._stagedCommit = stagedCommit;
+            return;
+        }
+        this._log("staged commit does not exist");
     }
 
+
```

Comment:
If we usually use a single line to delimit functions, please remove this extra line.

Reply:
Done.